docs/drive-management.md

```diff
@@ -0,0 +1,111 @@
+          name: vdb
+          size: 536870912
+          make: ""
+          select: "yes"
+    - name: node1
+      drives:
+        - id: 252:16$gGz4UIuBjQlO1KibOv7bZ+kEDk3UCeBneN/UJdqdQl4=
+          name: vdb
+          size: 536870912
+          make: ""
+          select: "yes"
+
+```
+
+2. Run `init` command.
+
+   The `init` command creates a request to add the selected drives in the YAML file generated using the `discover` command. As this process wipes out all data on the selected drives, wrong drive selection will lead to permanent data loss. Below is an example of `init` command:
+
+```sh
+$ kubectl directpv init drives.yaml
+
+ ███████████████████████████████████████████████████████████████████████████ 100%
+
+ Processed initialization request 'c24e22f5-d582-49ba-a883-2ce56909904e' for node 'master' ✔
+ Processed initialization request '7e38a453-88ed-412c-b146-03eef37b23bf' for node 'node1' ✔
+
+┌──────────────────────────────────────┬────────┬───────┬─────────┐
+│ REQUEST_ID                           │ NODE   │ DRIVE │ MESSAGE │
+├──────────────────────────────────────┼────────┼───────┼─────────┤
+│ c24e22f5-d582-49ba-a883-2ce56909904e │ master │ vdb   │ Success │
+│ 7e38a453-88ed-412c-b146-03eef37b23bf │ node1  │ vdb   │ Success │
+└──────────────────────────────────────┴────────┴───────┴─────────┘
+```
+
+Refer to the [discover command](./command-reference.md#discover-command) and the [init command](./command-reference.md#init-command) for more information.
+
+## List drives
+To get information of drives from DirectPV, run the `list drives` command. Below is an example:
+
+```sh
+$ kubectl directpv list drives
+┌────────┬──────┬──────┬─────────┬─────────┬─────────┬────────┐
+│ NODE   │ NAME │ MAKE │ SIZE    │ FREE    │ VOLUMES │ STATUS │
+├────────┼──────┼──────┼─────────┼─────────┼─────────┼────────┤
+│ master │ vdb  │ -    │ 512 MiB │ 506 MiB │ -       │ Ready  │
+│ node1  │ vdb  │ -    │ 512 MiB │ 506 MiB │ -       │ Ready  │
+└────────┴──────┴──────┴─────────┴─────────┴─────────┴────────┘
+```
+
+Refer to the [list drives command](./command-reference.md#drives-command) for more information.
+
+## Label drives
+Drives are labeled to set custom tagging which can be used in volume provisioning. Below is an example:
+```sh
+# Set label 'tier' key to 'hot' value.
+$ kubectl directpv label drives tiet=hot
+
+# Remove label 'tier'.
+$ kubectl directpv label drives tier-
+```
+
+Refer to the [label drives command](./command-reference.md#drives-command-1) for more information.
+
+## Replace drive
+Replace a faulty drive with a new drive on a same node. In this process, all volumes in the faulty drive are moved to the new drive then faulty drive is removed from DirectPV. Currently, DirectPV does not support moving data on the volume to the new drive. Use [replace.sh](./tools/replace.sh) script to perform drive replacement. Below is an example:
+```sh
+# Replace 'sdd' drive by 'sdf' drive on 'node1' node
+$ replace.sh sdd sdf node1
```

Comment:
Can we provide the step by step guide as comments in the script? customers can use this incase if the script doesn't work.

Reply:
I added those steps initially and removed them later. As commands have to be run conditionally, it is high chance of copy/paste error. If the script doesn't work, steps also won't work. If someone wants to understand what steps are carried out, reading `release.sh` script is quite straight-forward.